Frends.Community.Email.Tests/ExchangeEmailTests.cs

```diff
@@ -0,0 +1,651 @@
+            {
+                MaxEmails = 1,
+                DeleteReadEmails = false,
+                GetOnlyUnreadEmails = false,
+                MarkEmailsAsRead = false,
+                IgnoreAttachments = false,
+                GetOnlyEmailsWithAttachments = true,
+                AttachmentSaveDirectory = dirPath,
+                OverwriteAttachment = true
+            };
+
+            Directory.CreateDirectory(dirPath);
+            var result = await ReadEmailTask.ReadEmailFromExchangeServer(settings, options, new CancellationToken());
+            Assert.IsTrue(File.Exists(result[0].AttachmentSaveDirs[0]));
+            Assert.IsTrue(Directory.GetFiles(dirPath).Length == 1);
+            result = await ReadEmailTask.ReadEmailFromExchangeServer(settings, options, new CancellationToken());
+            Assert.IsTrue(File.Exists(result[0].AttachmentSaveDirs[0]));
+            Assert.IsTrue(Directory.GetFiles(dirPath).Length == 1);
+            Directory.Delete(dirPath, true);
+            await DeleteMessages(subject);
+        }
+
+        [Test]
+        public async Task SendEmailWithPlainTextTest()
+        {
+            var input = new ExchangeInput
+            {   
+                To = _username,
+                Message = "This is a test message from Frends.Commmunity.Email Unit Tests.",
+                IsMessageHtml = false,
+                MessageEncoding = "utf-8",
+                Subject = "Email test - PlainText"
+            };
+
+            var server = new ExchangeServer
+            {
+                Username = _username,
+                Password = _password,
+                AppId = _applicationID,
+                TenantId = _tenantID
+            };
+
+            var result = await EmailTask.SendEmailToExchangeServer(input, null, server, new CancellationToken());
+            Assert.IsTrue(result.EmailSent);
+            await DeleteMessages(input.Subject);
+        }
+
+        [Test]
+        public async Task SendEmailWithPlainTextToMultipleUsingSemicolonTest()
+        {
+            var input = new ExchangeInput
+            {
+                To = _username + ";" + _username,
+                Message = "This is a test message from Frends.Commmunity.Email Unit Tests.",
+                IsMessageHtml = false,
+                MessageEncoding = "utf-8",
+                Subject = "Email test - MultipleSemiColon"
+            };
+
+            var server = new ExchangeServer
+            {
+                Username = _username,
+                Password = _password,
+                AppId = _applicationID,
+                TenantId = _tenantID
+            };
+
+            var result = await EmailTask.SendEmailToExchangeServer(input, null, server, new CancellationToken());
+            Assert.IsTrue(result.EmailSent);
+            await DeleteMessages(input.Subject);
+        }
+
+        [Test]
+        public async Task SendEmailWithPlainTextToMultipleUsingColonTest()
```

Comment:
Is plaintext part important in this test?
Also I think you mean a comma, not a colon.